### ISSUE TITLE
Document how to write + publish + share third-party components

### ISSUE THREAD

**Issue by cvan** [aframevr/aframe-core#705]
1. Document how to use the methods we expose on `aframeCore`/`AFRAME` (e.g., `aframeCore.registerComponent`)
2. Publish the code on GitHub, deploy online somewhere (e.g., GH Pages, CodePen, Dropbox)
3. Open a PR against https://github.com/aframevr/awesome-aframe to get a link added to it
4. Write a case study on http://aframevr.tumblr.com (if you're so willing!)

**Comment by cvan** [comment]
it's been asked a few times on Slack too. again here on PH: https://www.producthunt.com/tech/aframe#comment-200325

**Comment by cvan** [comment]
also see aframevr/aframe#298

**Comment by ngokevin** [comment]
https://github.com/ngokevin/aframe-component-boilerplate

https://aframe.io/docs/core/component.html

+ I will write a blog post soon

**Comment by cvan** [comment]
and feel free to link in the Docs to your `aframe-component-boilerplate` or even move it under the `aframevr` org (if you'd like, up to you)

**Comment by ngokevin** [comment]
Yeah I might move it under the org so it gets more attention. Probably likewise with the react stuff.

**Comment by ngokevin** [comment]
I might plan on waiting until 0.2.0 to start heavily writing guides because of the changes to the component API and schema that are landing.
